akka-http-core/src/test/scala/akka/http/impl/engine/client/ConnectionPoolSpec.scala

```diff
@@ -388,7 +388,7 @@ class ConnectionPoolSpec extends AkkaSpec("""
               case x ⇒ fail(x.toString)
             }.toMat(Sink.fold(0)(_ + _))(Keep.both).run()
 
-        Await.result(idSum, 30.seconds) shouldEqual N * (N + 1) / 2
+        Await.result(idSum, 35.seconds) shouldEqual N * (N + 1) / 2
```

Comment:
True, that spec contains several tests with long await thresholds. We saw it fail at 30.4s so we could bump it to 31. Else I would prefer to remove that commit from the PR and look at the issue in a separate PR.

I'm also wondering if it would be possible to get rid of most of the test timeout failures. In most tests I've seen the encoded timeouts are AFAIK not related with correctness, for example request/response rendering.